Gemfile

```diff
@@ -3,6 +3,7 @@ gemspec
 
 # Fork to allow for a recent version of multipart-post.
 gem 'pedump', git: 'https://github.com/ksubrama/pedump', branch: 'patch-1'
+gem 'pry'
```

Comment:
This is for debugging only (thus putting in the gemfile so people don't get it when they `gem install omnibus`).